### DISCUSSION TITLE
No schedule settings on Admin page

### DISCUSSION THREAD

**Discussion by JonWhiteFang** [alexjustesen/speedtest-tracker#1604] [Q&A]
Hi, 
I do not have a schedule page on my settings so can only run this manually.
How do I set it up to be on a schedule?

<img width="1350" alt="image" src="https://github.com/user-attachments/assets/07ce97eb-1325-4344-b479-d8b8f1ccaf8a">

**Comment by jiriteach** [comment]
Read the release notes - been removed. Use environmental variables instead.

**Comment by seh2000** [comment]
SPEEDTEST_SCHEDULE=5 */1 * * *
This makes the test run automatically each hour
